great_expectations/data_context/migrator/cloud_migrator.py

```diff
@@ -143,81 +149,142 @@ def migrate_validation_result(
 
     def _migrate_to_cloud(self, test_migrate: bool) -> None:
         """TODO: This is a rough outline of the steps to take during the migration, verify against the spec before release."""
-        self._warn_if_test_migrate()
+        self._print_migration_introduction_message()
+
         configuration_bundle: ConfigurationBundle = ConfigurationBundle(
             context=self._context
         )
-        self._warn_if_usage_stats_disabled(
-            configuration_bundle.is_usage_stats_enabled()
+        self._emit_warnings(
+            configuration_bundle=configuration_bundle, test_migrate=test_migrate
+        )
+        self._print_configuration_bundle_summary(
+            configuration_bundle=configuration_bundle
+        )
+
+        serialized_bundle = self._serialize_configuration_bundle(
+            configuration_bundle=configuration_bundle
         )
-        self._print_configuration_bundle(configuration_bundle)
+        serialized_validation_results = self._prepare_validation_results(
+            serialized_bundle=serialized_bundle
+        )
+
         if not test_migrate:
-            configuration_bundle_serializer = ConfigurationBundleJsonSerializer(
-                schema=ConfigurationBundleSchema()
-            )
-            configuration_bundle_response: AnyPayload = self._send_configuration_bundle(
-                configuration_bundle, configuration_bundle_serializer
+            self._send_configuration_bundle(serialized_bundle=serialized_bundle)
+            self._send_validation_results(
+                serialized_validation_results=serialized_validation_results,
```

Comment:
General cleanup just to tighten up the flow a bit - I think this is still a bit fluid as I'm concurrently working on the printing ticket

Reply:
I like it!